pennylane/transforms/dynamic_one_shot.py

```diff
@@ -451,35 +439,35 @@ def _gather_counts(measurement: CountsMP, samples, is_valid, postselect_mode=Non
 # pylint: disable=unused-argument
 @gather_non_mcm.register
 def _gather_samples(measurement: SampleMP, samples, is_valid, postselect_mode=None):
+    samples = qml.math.concatenate(samples) if isinstance(samples, (list, tuple)) else samples
```

Comment:
Is this line also necessary after we no longer squeeze singleton? I'm thinking removal of ln457-458 as very expected but kinda puzzled by how an extra concatenation is required now

Reply:
Before. `stack` call was elsewhere in the source code
```
>>> x, y, z = np.array(1), np.array(0),  np.array(1)
>>> qml.math.stack([x,y,z])
array([1, 0, 1])
```
Now:
```
>>> x, y, z = np.array([[1]]), np.array([[0]]), np.array([[1]])
>>> qml.math.concatenate([x,y,z])
array([[1],
       [0],
       [1]])
```

Reply:
Ah I see! Thanks for clarification